### ISSUE TITLE
[Bug] db files appending tilde

### ISSUE THREAD

**Issue by CodeSpent** [getinsomnia/insomnia#1029]
- Insomnia Version: 5.16.6
- Operating System: Win10 x64

## Details

Every once in a while Insomnia seems to switch between looking for `myFile.db` format to looking for `myFile.db~` format. No pattern really seen, but occurs on both of my machines resulting in being unable to access saved requests. Renaming the files works temporarily, until Insomnia starts looking for the opposing naming convention.

![image](https://user-images.githubusercontent.com/7674344/42185367-a0318b38-7e16-11e8-9aac-fcfd4bbc2282.png)

**Comment by gschier** [comment]
Insomnia uses [NeDB](https://github.com/louischatriot/nedb) as it's local datastore. It looks like it uses temporary files in order to perform "crash-safe" writes. 

How large is the `insomnia.Request.db` file on your disk? I've haven't noticed anyone else report this error before so it might either be related to your storage device, the size of the file, or even a permissions issue.

**Comment by gschier** [comment]
FYI, the way crash-safe writes work is by first writing all data into a temporary file, then renaming the temporary file to overwrite the existing one. This is "crash-safe" because the destructive operation (renaming) is atomic, and thus cannot be affected by an application crash.

**Comment by CodeSpent** [comment]
The file size of `insomnia.Request.db` is 12KB so I don't believe its a size issue. I'm unsure if it could be permissions with my Microsoft account. I've experienced the issue on multiple machines, but both use the same Microsoft Account, When running as admin (a local account on each), I've been unable to recreate the issue, but after multiple reinstalls (deep uninstalls) the issue still persists once in a while on that standard Microsoft Account.

So I imagine the crash-safe temp files are appending the tilde to the end of the database file, but I can't seem to find any crashes in the logs that could explain why it'd do so. I've cleared my logs for now, and will report back the next time it happens.
